tests/parallel/treadafterwrite.nim

```diff
@@ -1,7 +1,9 @@
 discard """
   errormsg: "'foo' not disjoint from 'foo'"
   line: 23
-  disabled: "true"
+
```

Comment:
need to be careful with new lines introduced here, it'll throw the line number above off.